.github/workflows/release_branches.yml

```diff
@@ -648,3 +648,25 @@ jobs:
             echo "Error: $REPORT_LINK" >> $GITHUB_STEP_SUMMARY
             exit 1
           fi
+  LeakCheck:
+    if: ${{ !cancelled() }}
+    needs: FinishCheck
+    runs-on: [self-hosted, altinity-on-demand, altinity-style-checker-aarch64]
+    env:
+      PREFIX: ${{ github.event.pull_request.number || 0 }}/${{ github.event_name == 'pull_request' && github.event.pull_request.head.sha || github.sha }}
+    steps:
+      - name: Checkout scripts repo
+        uses: actions/checkout@v4
+        with:
+          repository: Altinity/actions
+          ref: a617d7c75203e08cfcfff0766b01ace3c58235e5
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y dpkg-dev rpm2cpio cpio
+      - name: Run leak check 1
+        run: |
+          python3 scripts/scan_s3_artifacts.py altinity-build-artifacts ${{ env.PREFIX }}
```

Comment:
We also need to add re to match `ROBOT_TOKEN` from here: https://github.com/Altinity/ClickHouse/pull/693/files#diff-5399cabc63b54fc614af664aabf2e13ece95190dbc6b2442d1becc25d98c87faR14